src/pages/Banner.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+
+interface Marks {
+  subject: string;
+  mark: number;
+}
+
+interface Student {
+  name: string;
+  id: string;
+  marks: Marks[];
+}
+
+interface ClassData {
+  name: string;
+  teacherName: string;
+  students: Student[];
+}
```

Comment:
I think we need to use this types in some other places, So it is better to keep a separate types file for these types and export from there.